src/views/Account/AccountCard/index.tsx

```diff
@@ -80,6 +115,17 @@ export const TotalBalanceAmount: React.FC<{ usersAddress: string }> = (props) =>
   return <CountUp countTo={Number(balancesData?.totalV4Balance)} />
 }
 
+export const ActiveBalance: React.FC<{ value: number }> = (props) => {
+  const { value } = props
+  const { t } = useTranslation()
+  return (
+    <span className='font-semibold text-xs'>
+      {t('active')}:&nbsp;$
```

Comment:
We could also just use `formatCurrencyNumberForDisplay` for simplicity. Once we add multiple currencies we'll thank ourselves for that. I find too many of the rolling numbers to be distracting.

Reply:
Resolved with 4cc42e9